### DISCUSSION TITLE
IP Block ( 76.76.21.142, 76.76.21.89, and 76.76.21.98 )

### DISCUSSION THREAD

**Discussion by 4freeline** [vercel/community#4833] [Help]
### Summary

It seems that we are not receiving responses from the official IPs of Samsung Electronics employees, 210.94.41.89 and 165.213.7.41, when accessing our website ([sdc-korea.com](http://sdc-korea.com/)) from IP addresses 76.76.21.142, 76.76.21.89, and 76.76.21.98. We suspect that there may be a blocking issue on the Vercel side.

I believe this phenomenon may be related to IP blocking by Vercel, but I have no idea where to contact or what steps to take. If anyone knows the process, please share it with me.

### Example

sdc-korea.com

### Steps to Reproduce

<img width="352" alt="image" src="https://github.com/vercel/community/assets/11920393/c88a3a04-5e1b-4147-9a8b-fbe2b3793c1f">

**Comment by amyegan** [comment]
I don't know about a general block on Vercel's side, though an individual site could have set up specific IP blocking of their own. If it's only happening for one site, that may be the cause. 

If it's also happening for other sites, I have seen similar problems in the past when a company has network rules that block access to `*.vercel.app` sites. Please check with your network admin to see if they need to create an exception to allow access. 🙏
